package.json

```diff
@@ -44,6 +44,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.6.1",
+    "eslint_d": "^5.2.2",
```

Comment:
what this one does?

Reply:
Make eslint work with my setup and ensures we all use the same version of eslint_d